compiler/src/main/java/com/squareup/anvil/compiler/codegen/PsiUtils.kt

```diff
@@ -456,3 +453,16 @@ fun KtClassOrObject.injectConstructor(injectAnnotationFqName: FqName): KtConstru
     )
   }
 }
+
+fun KtClassOrObject.requireClassDescriptor(module: ModuleDescriptor): ClassDescriptor {
```

Comment:
Really liking the cleanup/creation of extensions in your PRs. :chefs-kiss:

Reply:
🙏